serve/mlc_serve/engine/engine_common.py

```diff
@@ -132,6 +133,31 @@ def detokenize_incrementally(
     return delta
 
 
+def logprob_detokenize(tokenizer: TokenizerP, logprob_info: Optional[LOGPROBS_TYPE]) -> Optional[LOGPROBS_TYPE]:
+    """Detokenize logprob information"""
```

Comment:
This sentence doesn't make sense

Reply:
Changed to "Detokenize top tokens in logprob information", does it make more sense?

Reply:
sounds good